check_file_ages_in_dirs

```diff
@@ -285,7 +301,11 @@ do
   # Check if dir exists.
   full_path=${base_dir}${dir}
   if [ -d "$full_path" ]; then
-    file_list=`ls -t $full_path`
+    if [ "$ignore" ]; then
```

Comment:
Can you add the -n operator to this test? I think that's more clear in the code.